### ISSUE TITLE
Does not seem to work with VLC 3 

### ISSUE THREAD

**Issue by Salamandar** [exebetche/vlsub#153]
I know it's a bit ahead-of-time and VLC 3.0 isn't released yet, but I like to live dangerously. 
And VLSub does not seem to work here. It appears in the Extensions list, but nothing is added in View>.

**Comment by enedil** [comment]
I've got VLC 3.0 and although I see vlsub in View> it doesn't show any additional window.

**Comment by pcasaretto** [comment]
I can confirm VLC 3.0.0 built from current git works with vlsub 0.10.2 .

**Comment by ThatLurker** [comment]
@pcasaretto On what OS?

**Comment by pcasaretto** [comment]
macOS Sierra 10.12.6

**Comment by ThatLurker** [comment]
The issues that many get from the git version is caused by it not having the required tool to unzip the subtitle file

**Comment by ThatLurker** [comment]
It might have been fixed in the latest vlc git version

**Comment by pcasaretto** [comment]
Actually, I've been having trouble with unzipping as well. VLSub is able to find the subtitle but when it is time to unzip the file, an zero size file is created and the extensions hangs. I usually fix this manually.

**Comment by ThatLurker** [comment]
Yeah, its caused by the git version of vlc. Its all good on the release version tough
